### DISCUSSION TITLE
general question about axum and rust

### DISCUSSION THREAD

**Discussion by liy9426** [tokio-rs/axum#1566] [General]
This is probably not relevant to axum but more of a rust and webpack question. 
I was wondeirng is it possible to wrap the axum service into a webpack plugin or something similar, then proxy webpack-dev-server requests to axum service? Because I'd like the service to be available for the whole team but rather not asking everyone to start the service from their dev client manually. It's like a double proxy process during web dev and integrate into webpack framework. I can't put the service on a server either.
Appreciate if someone could provide some help or point out better way to do it

**Comment by davidpdrsn** [comment]
Sorry but I know next to nothing about webpack so you're probably better off asking somewhere else.
